vep/__init__.py

```diff
@@ -279,15 +281,45 @@ def install_build_tools(self, pip):
             else:
                 pip('install', "%s==%s" % (tool, version), _out=print_line)
 
+    def _pip_requirements_filename(self, path=''):
+        """
+        Returns filename of pip requirements file, & verifies that the file exists.
+
+        :param path: Path to dir to search. Default it current dir.
+        :type path: str
+        :raises VEPackagerError
+        :return: Filename of pip requirements
+        :rtype: str
+        """
+        if self.args.pip_requirements:
+            filenames = list(self.args.pip_requirements)
```

Comment:
You should be able to use `nargs='*'` or `action='append'` to get a list from the start.

Reply:
I’m not trying do any of that: I’m just just making a list that contains one string.

The reason I want a list is so the logic below can iterate over a list. The list will either be `['my-snowflake-file.pip']` or `['requirements.txt', 'requirements.pip']`.